packages/office-ui-fabric-react/src/components/Overlay/Overlay.styles.ts

```diff
@@ -0,0 +1,52 @@
+import { IOverlayStyleProps, IOverlayStyles } from './Overlay.types';
+import {
+  IStyle,
+  ITheme,
+  HighContrastSelector,
+} from '../../Styling';
+
+export const getStyles = (
+  props: IOverlayStyleProps
+): IOverlayStyles => {
+  const {
+    className,
+    theme,
+    isNone,
+    isDarkThemed,
+  } = props;
+
+  const { palette, semanticColors } = theme;
+
+  return ({
+    root: [
+      'ms-Overlay',
+      {
+        backgroundColor: palette.whiteTranslucent40,
+        top: 0,
+        right: 0,
+        bottom: 0,
+        left: 0,
+        position: 'absolute',
+
+        selectors: {
+          [HighContrastSelector]: {
+            border: '1px solid WindowText',
```

Comment:
Is WindowText a valid choice here? This may be High Contrast Mode specific, but I just imagine it would be better to use a theme color.

Reply:
Yup, it's HighContrast specific.